src/aspire-dashboard/8.0/cbl-mariner-distroless/amd64/Dockerfile

```
@@ -0,0 +1,31 @@
+ARG REPO=mcr.microsoft.com/dotnet/aspnet
+
+# Installer image
+FROM mcr.microsoft.com/cbl-mariner/base/core:2.0 AS installer
+
+RUN tdnf install -y \
+        ca-certificates \
+        unzip \
+    && tdnf clean all
+
+# Retrieve Aspire Dashboard
+RUN dotnet_aspire_version=8.0.0-preview.3.24075.10 \
+    && curl -fSL --output dotnet-aspire.tar.gz https://dotnetbuilds.azureedge.net/public/aspire/$dotnet_aspire_version/aspire-dashboard-linux-x64.zip \
```

NIT, should output be dotnet-aspire.zip?
```suggestion
    && curl -fSL --output dotnet-aspire.zip https://dotnetbuilds.azureedge.net/public/aspire/$dotnet_aspire_version/aspire-dashboard-linux-x64.zip \
```

---

I'm a big fan of `-O` with `curl` when the filename isn't too long.